docs/tractus-x-release.md

```diff
@@ -0,0 +1,131 @@
+## Intro
+
+Tractus-X operates on a quarterly release cycle. Each quarter, a new Tractus-X release following CalVer is released. Each Tractus-X release contains multiple products product_release.md.
+
+Once per year, Tractus-X releases a major release, making the remaining three releases minor releases. A major release may contain critical breaking changes that have a major impact on data space participants, such as changes to enablement services.
+
+A minor release contains backward compatible functionality. In addition, patch versions provide backward compatible bug and security fixes.
+
+The overarching Tractus-X releases are coordinated by the release management group (SIG Release). This team collaborates with the currently released versions of Tractus-X products to formulate the overarching release.
+
+As mentioned in [planning.md](planning.md) the Tractus-X project follows an overarching qualification process that all official quarterly Tractus-X releases adhere to. This process applies E2E activities for testing and security for all relevant release participants in compound, which stakeholders can build on. Each product can/shall be released on demand on a higher cadence, see [product_release.md].
+
+
+## Scope / Content
+
+The Tractus-X release process aims to ensure compatibility of Tractus-X products and defines common quality criteria.
+
+The desired enhancement of the dataspace functionality is planned in the [open refinement and planning](open-refinement-and-planning.md) sessions.
+
+Past release documentation can be found in [eclipse-tractusx/tractus-x-release](https://github.com/eclipse-tractusx/tractus-x-release).
+
+Note: In every quarterly TX release, we shall expect evolutions (new versions) of some products (see [product_release.md]). But also some other products may NOT face a change/up-revision in the given iteration. Nevertheless, those products (“carry-over”) are still essential parts of the release bundle.
+
+Products **must** opt-in to the process in order to be included in a Tractus-X release. (see product release.md)
+
+
+## Process abstract (future state; or “to-be”)
+
+- Planning
+- Feature Work and Product Release
+- Quality assurance in parallel
+- Opt-In for participation in TX Release
+- Completion of Q-Assurance
+- TX Release
+
+Once feature work on product level is initiated, the quality assurance of the foreseen release package runs in parallel. The vehicle of Umbrella helm charts is used to continuously test pre-defined product bundles. Automatically and environment-less.
+
+Reference onion rings: (add URL)
```

Comment:
this helpful graph shall soon be published within a document of the CX Association => we can add the URL as soon as available

Reply:
still trying to obtain URL

Reply:
@RolaH1t url still missing; i keep it removed. We can do a patch if you get the url